src/qt/forms/debugwindow.ui

```diff
@@ -925,6 +925,9 @@
              </property>
              <property name="textElideMode">
               <enum>Qt::ElideMiddle</enum>
+              </property>
```

Comment:
You're introducing this property within the `textElideMode` property. 

You should be introducing this after line `928` on the original file, not after line `927`. That way you don't have to introduce this closing tag to then introduce the property. 

Also, the indentation is off here.

Reply:
Just trying to edit .ui files without qt-creator - the diff junk qt-creator makes is annoying.

Reply:
I'm using Qt Designer to edit UI files.

Yeah, in most cases diff junk is unavoidable, but it is easy for me to revert it in Sublime Merge :)